frontend/src/components/Utils/filter-input.js

```diff
@@ -0,0 +1,87 @@
+import * as firebase from 'firebase/app';
+
+import { getCurUserEmail, getUidFromUserEmail } from './temp-auth-utils.js'
```

Comment:
Heads up that the name of the second function will likely use the ones as described in #63.

Also I like possibly using `getCurrUserEmail` rather than my current `getUserEmail` since it's clearer, so I might update my auth utils for that.

Reply:
Sounds good. Changed the two functions to match #63.